video_llama/models/video_llama.py

```diff
@@ -134,6 +134,8 @@ def encode_videoQformer_visual(self, image):
         image = einops.rearrange(image, 'b c t h w -> (b t) c h w')
         with self.maybe_autocast():
             # embed image features with blip2, out: (b t) q h
+            if device == torch.device("cpu"):
+                self.vit_to_cpu()
```

Comment:
All we needed to change to avoid the `*** RuntimeError: Input type (float) and bias type (c10::Half) should be the same` on cpu